components/trial_benefits_modal/trial_benefits_modal.tsx

```diff
@@ -65,48 +59,50 @@ const TrialBenefitsModal = ({
     const trialLicenseDuration = (1000 * 60 * 60 * 24 * 30) + (1000 * 60 * 60 * 8);
     const trialEndDate = moment.unix((Number(license?.ExpiresAt) || new Date(Date.now()).getTime() + trialLicenseDuration) / 1000).format('MMMM,DD,YYYY');
 
+    const buttonLabel = formatMessage({id: 'learn_more_trial_modal_step.learnMoreAboutFeature', defaultMessage: 'Learn more about this feature.'});
+
     const steps: TrialBenefitsModalStepProps[] = useMemo(() => [
         {
-            id: 'guestAccess',
-            title: formatMessage({id: 'trial_benefits.modal.guestAccessTitle', defaultMessage: 'Determine user access with Guest Accounts'}),
-            description: formatMessage({id: 'trial_benefits.modal.guestAccessDescription', defaultMessage: 'Collaborate with users outside of your organization while tightly controlling their access channels and team members.'}),
+            id: 'useSso',
+            title: formatMessage({id: 'trial_benefits.modal.useSsoTitle', defaultMessage: 'Use SSO (with OpenID, SAML, Google, O365)'}),
+            description: formatMessage({id: 'trial_benefits.modal.useSsoDescription', defaultMessage: 'Sign on quickly and easily with our SSO feature that works with OpenID, SAML, Google, and O365.'}),
             svgWrapperClassName: 'guestAccessSvg',
             svgElement: (
                 <GuestAccessSvg
                     width={400}
                     height={180}
                 />
             ),
-            pageURL: ConsolePages.GUEST_ACCESS,
-            buttonLabel: formatMessage({id: 'benefits_trial.modal.learnMore', defaultMessage: 'Learn More'}),
+            pageURL: 'https://docs.mattermost.com/onboard/sso-saml.html',
```

Comment:
Could you please add all new url to the doc links in constants? : https://github.com/mattermost/mattermost-webapp/blob/master/utils/constants.tsx#L932